modules/user_auth/README.md

```diff
@@ -0,0 +1,39 @@
+# user_auth
+
+User management and Identity / Access Proxy using Ory Kratos and Oathkeeper 
+
+## Notes
+
```

Comment:
How about something about how it will be created in the kubernetes cluster referenced by the provider passed to it, and requires a JWKS file to be created in the namespace previously, and how typically this is handled during the Zero apply step.

Reply:
👍 added a real note and  pre-req section 
```md
## Notes
This module uses helm-charts to create a user authentication suite (Kratos, Oathkeeper, Oathkeeper-measter) in your Kubernetes cluster through your Kubernetes Provider, to provide user identity and authentication.

## Pre-requisites
### jwks_secret_name
This is a Private key stored in secret-manager used to sign tokens for user sessions, it is created in the Zero apply step during [`pre-k8s.sh`](https://github.com/commitdev/zero-aws-eks-stack/blob/main/templates/scripts/pre-k8s.sh#L22-L32)

### auth database / secret 
This is a database and connection-credentials created for Kratos, it is created in the Zero apply step during [`pre-k8s.sh`](https://github.com/commitdev/zero-aws-eks-stack/blob/main/templates/scripts/pre-k8s.sh#L22-L32)

```